content/acra/_index.md

```diff
@@ -2,10 +2,25 @@
 weight: 1
 title: "Acra in a nutshell"
 bookFlatSection: true
-productMenuTitle: "acra"
+productMenuTitle: "Acra"
```

Comment:
mmmmmmmmmmmmmmmm have you actually tested this behaviour? because previous times changing to lowercase actually broke underline state of the menu.

Reply:
<img width="426" alt="acra" src="https://user-images.githubusercontent.com/2877920/150442243-1a0a935f-82bc-4cc1-add9-042b8e335bdb.png">

Reply:
sorry, I dont know about this feature, fixed